doc/DeveloperGuide.md

```diff
@@ -88,12 +90,35 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to list persons
+2. Addressbook shows a list of persons
+3. User requests to rename a certain tag to a new tag name
+3. Addressbook changes the tag of all users who had the certain tag to the new tag name <br>
+
+**Extensions**
+
+2a. The list is empty 
+
+> Use case ends
+
+3a. The tag the user wants to change is invalid
```

Comment:
- What happens when the new tag is already present in the address book?
- What happens if the new tag is same as the old tag?